h/views/activity.py

```diff
@@ -154,6 +154,29 @@ def user_annotation_count(aggregation, userid):
         result['stats'] = {
             'annotation_count': group_annotation_count,
         }
+
+        if self.group.type == 'private':
```

Comment:
In the places that you're setting string values in this dict, you might consider using `_`, as in `_('Members')` to make these i18n-ready.